openapi/v1/filesUser.yaml

```diff
@@ -0,0 +1,115 @@
+openapi: 3.1.0
+info:
+  description: |-
+    This swagger page describes the filesUser APIs.
```

Comment:
I think we need more details here

Reply:
same for filesCurator

Reply:
I'm afraid we can not write documentation without seeing the whole picture